src/aat/resources/features/F-066/S-180.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "must return negative response when request contains an invalid event trigger",
+	"_guid_": "S-180",
+  "_extends_": "F-066_Test_Data_Base",
+
+	"specs": [
+		"an active profile in CCD",
+		"contains an invalid event-trigger",
+		"has the 404 return code"
+	],
+  "request": {
+    "pathVariables": {
+      "cid": "[[DYNAMIC]]",
```

Comment:
could the case id be set as dynamic in the F-066_Test_Data_Base file so that it doesn't need to be explicitly set here?

Reply:
No, as explained above.

Reply:
Here, new case creation is mandatory as it will be validated before the event id.